pfcpiface/utils.go

```diff
@@ -23,6 +23,28 @@ func Set(b, flag Bits) Bits { return b | flag }
 //func Clear(b, flag Bits) Bits  { return b &^ flag }
 //func Toggle(b, flag Bits) Bits { return b ^ flag }
 //func Has(b, flag Bits) bool { return b&flag != 0 }
+func setUeipFeature(features ...uint8) {
+	if len(features) >= 3 {
+		features[2] = features[2] | 0x04
+	}
+}
+
+func has2ndBit(f uint8) bool {
+	return (f&0x02)>>1 == 1
```

Comment:
if bool return is not required then we can avoid >> 1 & == 1 check.
return (f & 0x02)

Reply:
we would have to add check for ==1 wherever we call has2ndBit. So will let it be as it is for now.